README.md

```diff
@@ -40,22 +40,23 @@ If you have specific questions about the project, our [Telegram Group](https://t
 * Support for Bitcoin Mainnet & Testnet
 * Support for custom user-defined derivation paths
 * On-demand receive address verification
+* Single-sig and multi-sig change address verification
 * User-configurable QR code display density
+* Taproot (P2TR) single-sig signing
 * Responsive, event-driven user interface
+* Customized Linux live-boot OS to allow MicroSD card removal
```

Comment:
The Networking, Bluetooth, USB and Swap modules have been removed from our Linux kernel. (Removed vs just disabled?)

Minimum surface-area? Linux OS which removes all Wifi and Bluetooth libraries at the OS Level.

Add as a separate/new features: 
Seedsigner_OS is a live-boot OS allowing your MicroSD card to be physically removed after starting, and _before_ your Private keys are input/scanned, to further improve the air-gapping around your Private keys.   
BIP-85 deterministic seed derivation is now supported.